docs/repo-docs/reference/run.mdx

```diff
@@ -121,13 +121,18 @@ turbo run build --concurrency=50%
 turbo run test --concurrency=5
 ```
 
-### `--continue`
+### `--continue [option]`
 
-Default: `false`
+Default: `none`
+
+Specify how `turbo` should handle current and pending tasks in the presence of an error (e.g. non-zero exit code from a task).
 
-Continue with task execution in the presence of an error (e.g. non-zero exit code from a task).
+- When `--continue` is `none` and an error occurs, `turbo` will cancel all tasks.
+- When `--continue` is `independent-tasks-only` and an error occurs, `turbo` will cancel all dependent tasks but continue running other independent tasks.
+- When `--continue` is `all` and an error occurs, `turbo` will continue running pending tasks (including dependents).
+- When `--continue` is specified without a value, it will default to `all`.
 
-When `--continue` is `true`, `turbo` will exit with the highest exit code value encountered during execution.
+In all cases, `turbo` will exit with the highest exit code value encountered during execution.
```

Comment:
actually i haven't verified if this is true in the `none` case 🤔

Reply:
based on both the previous and updated [continue.t output](https://github.com/vercel/turborepo/pull/10023/files#diff-950eab16b0680441205540b1c3080675aa365640711c5632d17708c99a9ae914), the exit code is not propagated upwards in any cases (`run build exited (1)` despite `exit 2`). so that sounds like an existing bug (or doc issue?). we could potentially address it in this PR and/or just remove this line from the docs, wdyt?

Reply:
Yes, `npm` has had odd behavior around exit codes from underlying scripts across versions. We do take the [highest exit code](https://github.com/vercel/turborepo/blob/main/crates/turborepo-lib/src/run/mod.rs#L489) from the process we spawn (or `1` if the error was not from a task exiting such as a task failing to start etc).